invoices/invoiceregistry_test.go

```diff
@@ -947,3 +948,36 @@ func TestInvoiceExpiryWithRegistry(t *testing.T) {
 		}
 	}
 }
+
+// TestSubscribeAll tests the all-invoices subscription.
+func TestSubscribeAll(t *testing.T) {
+	defer timeout()()
+
+	ctx := newTestContext(t)
+	defer ctx.cleanup()
+
+	// Add two invoices.
+	hash1 := lntypes.Hash{1}
+	_, err := ctx.registry.AddInvoice(testInvoice, hash1)
+	require.Nil(t, err)
```

Comment:
Nice to see `testify` making it's way to our tests :)
one nit: could use `assert.NoError(t, err)` here.

Reply:
fixed